src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -125,8 +125,9 @@ namespace backend { namespace opengl {
                                              *reinterpret_cast<GLuint*>(&mValues[stage][constant]));
                                 break;
                             case PushConstantType::Float:
-                                glUniform1f(location,
-                                            *reinterpret_cast<GLfloat*>(&mValues[stage][constant]));
+                                glUniform1f(
```

Comment:
Could this be done with maybe a memcpy into a std::array<float, kMaxPushConstants>? It is still technically a UB but at least on a different line so we can comment on this being required to avoid a strict aliasing warning.